mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -41,45 +42,45 @@ def requires(self):
 
     def run(self):
         logger.info("Run {task}".format(task=self.__class__.__name__))
-        # name in self.param.body['metadata']['name']
-        # namespace in self.param.body['metadata']['namespace']
-        # label in self.param.body['metadata']['label']
-        # annotations in self.param.body['metadata']['annotations']
-
-        # logger.info("name: {}".format(self.param.body['metadata']['name']))
-        # logger.info("namespace: {}".format(
-        #     self.param.body['metadata']['namespace']))
-        # logger.info("label: {}".format(self.param.body['metadata']['labels']))
-        # logger.info("annotations: {}".format(
-        #     self.param.body['metadata']['annotations']))
-        # logger.info("hostNetwork: {}".format(
-        #     self.param.spec.get('hostNetwork', 'False')))
-        # logger.info("hostIP: {}".format(self.param.body['status']['hostIP']))
-        # logger.info("phase: {}".format(self.param.body['status']['phase']))
-        # logger.info("podIPs: {}".format(self.param.body['status']['podIPs']))
 
         if "hostIP" not in self.param.body['status']:
             self.raise_temporary_error("Pod spec not ready.")
         spec = {
             'hostIP': self.param.body['status']['hostIP'],
             'name': self.param.body['metadata']['name'],
-            'namespace': self.param.body['metadata']['namespace'],
-            'tenant': '',
-            # TODO (Cathy) in case of arktos
-            # get VPC and net information from annotation
-            'vpc': OBJ_DEFAULTS.default_ep_vpc,
+            'type': COMPUTE_PROVIDER.k8s,
+            'namespace': self.param.body['metadata'].get('namespace', 'default'),
+            'tenant': self.param.body['metadata'].get('tenant', ''),
+            'vpc': self.param.body['metadata'].get('labels', {}).get(
+                    'arktos.futurewei.com/network', OBJ_DEFAULTS.default_ep_vpc),
             'net': OBJ_DEFAULTS.default_ep_net,
             'phase': self.param.body['status']['phase'],
-            # TODO (Cathy) in case of arktos get list of interfaces to create on
-            # the host (names)
+            'readiness': self.param.body['metadata'].get('annotations', {}).get(
+                    'arktos.futurewei.com/network-readiness', ''),
             'interfaces': [{'name': 'eth0'}]
         }
 
         logger.info("Pod spec {}".format(spec))
         spec['vni'] = vpc_opr.store_get(spec['vpc']).vni
         spec['droplet'] = droplet_opr.store_get_by_ip(spec['hostIP'])
-        # TODO (cathy): make sure not to trigger init or create simple endpoint
+
+        if 'arktos.futurewei.com/network' in self.param.body['metadata'].get('labels', {}):
```

Comment:
It's better to define constants for strings like "arktos.futurewei.com/network" or "arktos.futurewei.com/nic"